doc/release-notes-6107.md

```diff
@@ -0,0 +1,4 @@
+RPC changes
+-----------
+
+- `coinjoin stop` will now return an error if there is no CoinJoin mixing session to stop
```

Comment:
should it be really changed?

Reply:
Behaviour would match `abortrescan`, which returns `false` when a request is made and there isn't anything to abort ([source](https://github.com/dashpay/dash/blob/1f113587fb3d894fc64ef28e5ae5a912de8450c5/src/wallet/rpcdump.cpp#L184-L185)).